src/modules/areas.js

```diff
@@ -127,12 +128,15 @@ export function getAreas() {
 }
 
 // POST name, geostore ID
-export function saveArea(area) {
-  const url = `${API_BASE_URL}/area`;
-  const body = new FormData();
-  body.append('name', area.name);
-  body.append('geostore', area.geostore);
-  return (dispatch, state) => {
+export function saveArea(area, node) {
+  return async (dispatch, state) => {
+    const url = `${API_BASE_URL}/area`;
+    const body = new FormData();
+    const blob = await domtoimage.toBlob(node);
```

Comment:
check the dom-to-image params for height and width:
https://github.com/tsayen/dom-to-image#height-width
Maybe it can fix the gray borders and shown attribution problem